build-info-extractor/src/testFixtures/java/org/jfrog/build/IntegrationTestsBase.java

```diff
@@ -74,7 +76,7 @@ public void init() throws IOException {
             inputStream.close();
         }
 
-        url = readParam(props, "url");
+        url = Optional.ofNullable(readParam(props, "url")).orElse("");
```

Comment:
It looks like the new code is a dead one. `readParam` fails the test if the returned value is null.

Reply:
True, but there is still a warning that url might be null. This was added to make sure it isn't and remove the warning.